std/c/dragonfly.zig

```diff
@@ -0,0 +1,122 @@
+extern "c" threadlocal var errno: c_int;
+pub fn _errno() *c_int {
+    return &errno;
+}
```

Comment:
Could the definitions in this file be shared with freebsd.zig? The only difference is the errno definition, isn't it?

Reply:
In dragonfly __error is inlined, so that symbol wasn't available to c/dragonfly.zig. Could freebsd use c/dragonfly.zig though?? It sounds possible

Reply:
I know from some previous projects that e.g. dragonfly's kqueue implementation has more/different flags.

Reply:
Thanks for the heads up. This file is now auto generated, and no-longer copied from std/c/freebsd.zig